utils/config_file.hh

```diff
@@ -62,6 +63,12 @@ template <typename T>
 extern const config_type config_type_for;
 
 class config_file {
+    static thread_local unsigned s_shard_id;
+    struct any_value {
```

Comment:
Is this intended to be anything more than cloneable?  If so, please describe the intent in a blurb.  If not, please consider renaming it `cloneable` and moving it to a more general location.

Reply:
Yes, it gains an update_from() later on.

How can I view the lines below the one you quote in this conversation?

Reply:
> Yes, it gains an update_from() later on.

Sorry, I managed to miss that. :(
 
> How can I view the lines below the one you quote in this conversation?

Assuming you're viewing the comment on the PR [webpage](https://github.com/scylladb/scylla/pull/4559) (the "Conversation" tab), try clicking on the file name above it:
![image](https://user-images.githubusercontent.com/1635074/59720042-27727f00-921e-11e9-9373-f655f1f01f3a.png)

Reply:
Yes, that works well, thanks.